services/src/test/java/io/grpc/services/BinaryLogTest.java

```diff
@@ -203,6 +260,192 @@ public void configBinLog_ignoreDuplicates_method() throws Exception {
     assertEquals(HEADER_256, factory.getLog("p.other2/m"));
   }
 
+  @Test
+  public void callIdToProto() {
+    assertEquals(
```

Comment:
CALL_ID should be defined inside the test.  As written, I can't read the test to know if its correct.  I have to scroll elsewhere to see it's right.   For example How do I know where 0x1112131415161718L came from?

Reply:
I was intending to use CALL_ID for the next diff but I agree right now it's not buying us anything. Moved to test method body.